tomcat-serverxml-tool.sh

```diff
@@ -4,7 +4,7 @@ JAR=/usr/lib/tomcat/serverxmltool.jar
 SERVERXML=/etc/tomcat/server.xml
 
 function usage {
-    echo "Usage: <script> file.xslt [arg=value]..."
+    echo "Usage: tomcat-serverxml-tool.sh file.xslt [arg=value]..."
```

Comment:
TBH, how this works is a bit obscure to me.

I understand what the script does, but not sure how the xslt should look like and what's the relationship with the arg/value pair. Should we maybe provide an example, at least at the README?

Reply:
make sense. I'll provide an example

Reply:
What's your opinion @mbussolotto?